Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1680,6 +1689,10 @@ inline void BuilderCustom::applyInitialFontVariantNumeric(BuilderState& builderS
 
 inline void BuilderCustom::applyValueFontVariantNumeric(BuilderState& builderState, CSSValue& value)
 {
+    if (is<CSSPrimitiveValue>(value) && CSSPropertyParserHelpers::isSystemFontShorthand(downcast<CSSPrimitiveValue>(value).valueID())) {
+        applyInitialFontVariantLigatures(builderState);
```

Comment:
s/applyInitialFontVariantLigatures/applyInitialFontVariantNumeric ?

Reply:
Oh no, big mistake! Must fix this.